src/services/resource-finder.js

```diff
@@ -31,12 +27,7 @@ function ResourceFinder(model, params, withIncludes) {
       conditions.include = getIncludes();
     }
 
-    if (schema.isCompositePrimary) {
-      conditions.where = compositeKeysManager
-        .getRecordConditions(params.recordId);
-    } else {
-      conditions.where[schema.idField] = params.recordId;
-    }
+    conditions.where = new CompositeKeysManager(model).getRecordsConditions([params.recordId]);
```

Comment:
The naming is a little bit strange now that you also include non composite key inside the manager 🤔

Reply:
OK
I'll rename it to PrimaryKeyManager